packages/patternfly-4/react-core/src/components/DataList/DataListAction.js

```diff
@@ -5,13 +5,16 @@ import styles from '@patternfly/patternfly/components/DataList/data-list.css';
 import { EllipsisVIcon } from '@patternfly/react-icons';
 import { Button } from '../Button';
 
-const DataListAction = ({ className, id, 'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledBy, ...props }) => (
-  <div className={css(styles.dataListAction, className)} {...props}>
-    <Button variant="plain" id={id} aria-labelledby={ariaLabelledBy} aria-label={ariaLabel}>
-      <EllipsisVIcon />
-    </Button>
-  </div>
-);
+class DataListAction extends React.Component {
+  render() {
+    const {className, id, 'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledBy, rowid, ...rest} = this.props;
+    return <div className={css(styles.dataListAction, className)} {...rest}>
+      <Button variant="plain" id={id} aria-labelledby={ariaLabelledBy} aria-label={ariaLabel}>
```

Comment:
can you import and use the ButtonVariant object from Button and use that instead of using "plain".